heartbeat/azure-events.in

```diff
@@ -24,7 +24,31 @@ import ocf
 
 ##############################################################################
 
+def detect_distro():
```

Comment:
I guess we should move this to ocf. Maybe just ocf.distro()?

Reply:
Yeah, that would make sense. I can add it in a separate PR.

Reply:
Sure. Then you're free to merge it if you're done.

Reply:
Perfect, thanks.